clang/lib/Sema/SemaStmt.cpp

```diff
@@ -436,19 +437,68 @@ StmtResult Sema::ActOnCompoundStmt(SourceLocation L, SourceLocation R,
       DiagnoseEmptyLoopBody(Elts[i], Elts[i + 1]);
   }
 
-  // Set the checked scope specifiers for all statements that are part of this
-  // compound statement.
+  // Get the inferred checked scope specifier for this compound statement.
   CheckedScopeSpecifier InferredCSS = GetCheckedScopeInfo();
+
+  bool isBundledBlk = BNDLoc.isValid();
```

Comment:
`isBundledBlk` --> `IsBundledBlk` to make the variable name case consistent in this file.